jenkins-slaves/ansible/roles/common/tasks/main.yml

```
@@ -150,6 +150,23 @@
   with_items:
     - "{{ maven_versions }}"
 
+### Ant ###
+-name: Download Ant archives
+  get url:
+    url: https://archive.apache.org/dist/ant/binaries/apache-ant-{{ant_version}}-bin.tar.gz
```

I think the spaces inside {{ and }} should be preserved:
```suggestion
    url: https://archive.apache.org/dist/ant/binaries/apache-ant-{{ ant_version }}-bin.tar.gz
```